client/src/components/Tickets/TicketCreateForm.jsx

```diff
@@ -156,8 +162,8 @@ function TicketCreateForm({
                   id="attachments"
                   isSubmitted={isSubmitting}
                   description="Error reports and screenshots can be helpful for diagnostics"
-                  maxSizeMessage="Max File Size: 3MB"
-                  maxSize={3145728}
+                  maxSizeMessage={maxSizeMessage}
```

Comment:
@jmartinez-tacc Also, just noting that we will want to test this to make sure the max size for APCD is a per-attachment limit, and not a total attachment limit, meaning that if the user attaches multiple files adding up to over 15MB it will still work. From Nathan's [comment](https://jira.tacc.utexas.edu/browse/FP-1951?focusedId=26738&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26738):

> If I recall correctly, the limit for ticket submission on RT was that the sum of all the attachments was what was key.  I'm curious if that is correct so maybe that should be tested.

Reply:
We can deploy this branch to APCD and test it there

Reply:
@rstijerina @nathanfranklin
`FileInputDropZoneFormField` do that validation per attachment, not a total attachment limit.
Sure, it would be better to test it to APCD.
![Screenshot 2023-01-11 at 11 27 54 AM](https://user-images.githubusercontent.com/62672123/211902069-a05244fd-10be-4d0f-a693-e4bdb47b14ee.png)

Reply:
@rstijerina , @jmartinez-tacc Ah, i think what the case is that RT can take the attachments but then fails on sending an email if the sum of all the attachments exceeds so many MB. 

so i think the 3mb limit on Core-Portal was just a reflection of that.  like a hack to hope that if we limit the size to 3mb maybe they won't actually attach enough files to cause this emailing issue.  the RT server for APCD might be different.  

i don't know how important email is but just wanted to share all that.

an example ticket is https://consult.tacc.utexas.edu/Ticket/Display.html?id=86364&results=7837dac0850d696b293a8c161a0a22d6.  i add 10 3mb files and it works. when i add 30mb files then the outgoing email fails.